pyvista/plotting/renderer.py

```diff
@@ -3565,7 +3562,8 @@ def add_legend_scale(
             Factor to scale label size relative to title size.
 
         label_format : str, optional
-            A printf style format for labels, e.g. '%E'.
+            A printf style format for labels, e.g. ``'%E'``.
+            See `old-string-formatting <https://docs.python.org/3/library/stdtypes.html#old-string-formatting>`_.
```

Comment:
Last comment, I promise :smile: I think we should make use of intersphinx if we can, so that if this URL is ever moved in the future we don't have to rely on a permanent redirect.

But I'm not sure how to do that... The intersphinx inventory contains this corresponding line:
```
old-string-formatting std:label -1 library/stdtypes.html#$ printf-style String Formatting
```
As opposed to, say,
```
set py:class 1 library/stdtypes.html#$ -
```
Do you think this means we could reference this as
```suggestion
            See :std:label:`old-string-formatting`.
```
? I don't speak sphinx well enough to make sure based on https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html#module-sphinx.ext.intersphinx.

Reply:
Fixed in 82209fc849bf65f68c199023597adf873692ef0c. Tried a few ways and this was the one that worked.